packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -48,6 +62,15 @@ const ActiveVisitsTable = () => {
   const [currentPageSize, setPageSize] = useState(config?.activeVisits?.pageSize ?? 10);
   const [searchString, setSearchString] = useState('');
 
+  const currentPathName: string = window.location.pathname;
+  const fromPage: string = getOriginFromPathName(currentPathName);
+
+  const handleNameClick = (event: MouseEvent, to: string) => {
```

Comment:
Why is this function included in ActiveVisitsTable? Shouldn't it be directly in PatientNameLink?